scintillometry/channelize/base.py

```diff
@@ -0,0 +1,65 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import operator
+
+from ..base.base import TaskBase
```

Comment:
Similarly, a bit many `base` - refactoring to be done after?

Reply:
As above.